docs/troubleshooting-guide/filters.md

```diff
@@ -0,0 +1,46 @@
+# My filters don't work
+
+You have tried to add a [filter widget][filter-widget-gloss] to your dashboard, but:
+
+- the question you want to connect the filter to doesn't show up,
+- the filter doesn't show a dropdown list of possible values when you use it, or
+- the filter removes all of the rows from the table no matter what value you set it to.
+
+If you have created a [linked filter][linked-filter-gloss], please see [this troubleshooting guide](./linked-filters.html) instead.
+
+## Does the question actually include variables?
```

Comment:
I would say here: "does the SQL question include variables?" as if the question is a GUI question then it's just knowing if the question has the field you want to filter in the question